static/app/components/replays/useReplaysCount.tsx

```diff
@@ -43,9 +43,23 @@ function useReplaysCount({groupIds, transactionNames, organization, project}: Op
       );
     }
     if (groupIds && groupIds.length) {
+      // The endpoint only supports 25, that's also the
+      // max rows that the issue list page will show.
+      const chunkSize = 25;
+
+      const chunks = Math.ceil(groupIds.length / chunkSize);
+      const conditions: string[] = [];
+      for (let i = 0; i < chunks; i++) {
+        conditions.push(
+          `issue.id:[${toArray(groupIds.slice(i * chunkSize, (i + 1) * chunkSize)).join(
+            ','
+          )}]`
+        );
+      }
```

Comment:
i think lodash also has a chunk function that we've got imported somewhere

Reply:
> > is it possible to filter out the ones that have already been fetched or only fetch from the recent additions? You shouldn't need to do this, there will only ever be 25 new items max. but usually its 3 or 4 items
> 
> @scttcper i agree we've got to do this, but when the page is polling we might also want to refresh the count too. For example a brand-new issue might have been caught once, but does not yet have a replay attached; so we'd want to poll to get that replay count to move from `0` to `1`. I guess I can request if the count is falsey to account for those `0`'s in there. On the other hand if the issues page isn't polling for new data then the extra logic i'd add wouldn't be used. I made a separate issue #42310

Said all that, but then did it anyway to prevent unbounded requests from going out over and over and over again.

Reply:
oh, also related to the chunking/parallel calls, those are coming out! 
https://github.com/getsentry/sentry/pull/42242#discussion_r1047835438